runtime/runc_jailer.go

```diff
@@ -353,36 +367,68 @@ func (j runcJailer) jailerCommand(containerName string, isDebug bool) *exec.Cmd
 }
 
 // overwriteConfig will set the proper default values if a field had not been set.
-//
-// TODO: Add netns
-func (j runcJailer) overwriteConfig(cfg *Config, socketPath, configPath string) error {
-	spec := specs.Spec{}
-	configBytes, err := ioutil.ReadFile(configPath)
-	if err != nil {
-		return err
-	}
+func (j *runcJailer) overwriteConfig(cfg *Config, machineConfig *firecracker.Config, socketPath, configPath string) error {
+	var err error
+	j.once.Do(func() {
```

Comment:
What is the purpose of the once here? The all operations inside the closure look have no side-effects.

Reply:
caches the config so multiple reads on the file do not occur

Reply:
But the function is called by BuildJailedMachine() -> BuildJailedRootHandler() (this one cloud be private) -> overwriteConfig(). Can we remove this one for now? I don't think this file read would be a hot spot.

Reply:
Was worried about the file system call, but if you think it isn't going to be a hot spot I can remove it.